### ISSUE TITLE
d3sq: translations many places could use id/scope to offer better accessibility

### ISSUE THREAD

**Issue by monkeyiq** [filesender/filesender#1766]
An update should be made to the 3.x series to improve accessibility of tabled data.

Add either an 'id' or a 'scope' attribute to this <th> tag.

Associating <table> headers, i.e. <th> elements, with their <td> cells enables screen readers to announce the header prior to the data. This considerably increases the accessibility of tables to visually impaired users.

**Comment by WebSpider** [comment]
Does this mean updates to poeditor as well, or just the html table definition?

**Comment by monkeyiq** [comment]
There are updates to poeditor. I'm looking at one of the pages where it is present in the template to see what a change looks like that sonarqube is happy with.

**Comment by monkeyiq** [comment]
s/there are/there will be/

**Comment by monkeyiq** [comment]
It seems SQ doesn't like the thead element
https://github.com/filesender/filesender/pull/1768/files

**Comment by monkeyiq** [comment]
I imagine that this could be treated as part of a larger task where accessibility is considered on the whole for the new UI3 user interface.
